protocol-methods.rst

```diff
@@ -1016,8 +1159,8 @@ Only the first :func:`server.version` message is accepted.
 
 **Example**::
 
-  server.version("Electrum 3.0.6", ["1.1", "1.2"])
+  server.version("Electron Cash 3.3.6", ["1.2", "1.4"])
```

Comment:
Unrelated change

Reply:
So?  Let's just add it -- :) Can also add Elects too here. :)

Reply:
Looks like a "this is the reference implementation" change to me, bad vibes. We have enough of wannabe reference implementations. How about making this fictional?

Reply:
We might as well advertise our software dude.  I can add ElectrsCash in there too.  Stop being so rigid.  Nobody besides Electron Cash is even using this protocol now and it only connects to 3 servers.  Jesus.

Anyway I pushed a commit to satisfy you.